index.html

```diff
@@ -38,8 +40,130 @@ <h2>I am a Software Developer</h2>
+          <ul>
+            <li>HTML</li>
+            <li>Bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <button>See Project</button>
+        </section>
+        <section class="card card2">
+          <h1>Professional Art<br>Printing data</h1>
+          <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Dolores ea laborum praesentium ad recusandae fugit deserunt explicabo quisquam voluptatum debitis nisi reprehenderit repellat, consequatur eaque esse quos earum molestiae? Ea?</p>
+          <ul>
+            <li>HTML</li>
+            <li>Bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <button>See Project</button>
+        </section>
+        <section class="card card3">
+          <h1>Professional Art<br>Printing data</h1>
+          <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Dolores ea laborum praesentium ad recusandae fugit deserunt explicabo quisquam voluptatum debitis nisi reprehenderit repellat, consequatur eaque esse quos earum molestiae? Ea?</p>
+          <ul>
+            <li>HTML</li>
+            <li>Bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <button>See Project</button>
+        </section>
+        <section class="card card4">
+          <h1>Professional Art<br>Printing data</h1>
+          <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Dolores ea laborum praesentium ad recusandae fugit deserunt explicabo quisquam voluptatum debitis nisi reprehenderit repellat, consequatur eaque esse quos earum molestiae? Ea?</p>
+          <ul>
+            <li>HTML</li>
+            <li>Bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <button>See Project</button>
+        </section>
+        <section class="card card5">
+          <h1>Professional Art<br>Printing data</h1>
+          <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Dolores ea laborum praesentium ad recusandae fugit deserunt explicabo quisquam voluptatum debitis nisi reprehenderit repellat, consequatur eaque esse quos earum molestiae? Ea?</p>
+          <ul>
+            <li>HTML</li>
+            <li>Bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <button>See Project</button>
+        </section>
+        <section class="card card6">
+          <h1>Professional Art<br>Printing data</h1>
+          <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Dolores ea laborum praesentium ad recusandae fugit deserunt explicabo quisquam voluptatum debitis nisi reprehenderit repellat, consequatur eaque esse quos earum molestiae? Ea?</p>
+          <ul>
+            <li>HTML</li>
+            <li>Bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <button>See Project</button>
```

Comment:
- [ ] Please also, implement the hover and active state on this button.
- [ ] Every button element should have a type property on it. Please fix this.